src/libs/rover/ray_generators/camera_generator.hpp

```diff
@@ -50,14 +50,14 @@
 namespace rover {
 
 // TODO: Rename this to vtkmcamera ray generator
-class ROVER_API CameraGenerator : public RayGenerator
```

Comment:
why remove the `ROVER_API` macro?

Reply:
That's fair; I didn't understand the context. We should take a pass through this to decide what should have the macro and what shouldn't.